themes/bootstrap3/templates/admin/feedback/home.phtml

```diff
@@ -0,0 +1,153 @@
+<?php
+// Set page title.
+$this->headTitle($this->translate('VuFind Administration - Feedback Management'));
+?>
+<div class="<?=$this->layoutClass('mainbody')?>">
+  <h2><?=$this->translate('Feedback Management')?></h2>
+
+  <?=$this->flashmessages()?>
+
+  <form class="form-feedback-list" action="<?=$this->url('admin/feedback')?>" method="get">
+    <div class="feedback-controls">
+      <label for="form_name">
+        <?=$this->translate('Form name')?>
+        <select name="form_name" id="form_name" class="form-control">
+          <option value="ALL"><?=$this->translate('All')?></option>
+          <?php foreach($this->uniqueForms as $formName):?>
+            <option value="<?=$this->escapeHtmlAttr($formName)?>"<?php if ($formName === ($this->params['form_name'] ?? null)): ?> selected="selected"<?php endif;?>>
+              <?=$this->escapeHtml($formName) ?>
+            </option>
+          <?php endforeach;?>
+        </select>
+      </label>
+      <label for="site_url">
+        <?=$this->translate('Site URL')?>
+        <select name="site_url" id="site_url" class="form-control">
+          <option value="ALL"><?=$this->translate('All')?></option>
+          <?php foreach($this->uniqueSites as $site):?>
+            <option value="<?=$this->escapeHtmlAttr($site)?>"<?php if ($site === ($this->params['site_url'] ?? null)): ?> selected="selected"<?php endif;?>>
+              <?=$this->escapeHtml($site) ?>
+            </option>
+          <?php endforeach;?>
+        </select>
+      </label>
+      <label for="status">
+        <?=$this->translate('Status')?>
+        <select name="status" id="status" class="form-control">
+          <option value="ALL"><?=$this->translate('All')?></option>
+          <?php foreach ($this->statuses as $status): ?>
+            <option value="<?=$this->escapeHtmlAttr($status)?>"<?php if ($status == ($this->params['status'] ?? null)): ?> selected="selected"<?php endif;?>>
+              <?=$this->transEsc('feedback_status_' . $status)?>
```

Comment:
Maybe it would make sense to use `<?=$this->transEsc('feedback_status_' . $status, [], $status)?>` to default to the raw status value if the translation string is missing.

Reply:
It does not harm. Done